src/ipc/CMakeLists.txt

```diff
@@ -29,3 +29,14 @@ else()
 	add_compile_definitions(data_structs PUBLIC XCC_TOOLS_VERSION="")
 endif()
 
+add_library(data_structs STATIC "")
+
+# and optimization settings
+get_optimization_flag()
+add_compile_definitions(data_structs PUBLIC CC_OPTIMIZE_FLAGS="${optimization_flag}")
+
+include_directories(../include)
```

Comment:
This shouldn't be here, if you have depedency on sof_options, you should already have it in include dirs